analysis/analysis_utils.py

```diff
@@ -7,6 +7,7 @@
 import csv
 import astropy.coordinates as astroCoords
 import astropy.units as u
+import pdb
```

Comment:
Should we remove pdb imports before moving things to master?

Reply:
Probably. Done.